dub.json

```diff
@@ -0,0 +1,17 @@
+{
+	"name": "arrayfire",
+	"authors": [
+		"Prasun Anand"
+	],
+  "libs": ["afcuda", "cusolver", "cudart", "cufft", "cublas"],
```

Comment:
@pavanky , Don't we need the [documentation](https://dlang.org/spec/ddoc.html) for `arraytfire-d`? I had left the comments there for documentation.